app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav.scss

```diff
@@ -339,7 +348,7 @@ $top-border: solid 2px $grey-3;
   list-style: none;
 
   .gem-c-step-nav--large & {
-    @include media(tablet) {
+    @include govuk-media-query($from: tablet) {
       padding-bottom: 20px;
```

Comment:
govuk-spacing (4)

Reply:
There are loads of explicit px definitions in this file that could be replaced with the govuk-spacing mixin. I don't see the benefit in changing them all - the only benefit would be if the spacing scale changed, but then that would break the component.